### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things I did for this project 
> Install react-router-dom and make use of it
Add navbar with the corresponding links 
Modify the App.js component so we can rout through our pages 
Style  the App

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+        <Switch>
+          <Route exact path="/">
+            <div className="App">
+              <Home />
 
-      </div>
+            </div>
+          </Route>
+          <Route exact path="/calculator">
+            <Calculator />
```
Comment (Mihndim2020) [inline src/App.js:21]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", Please make sure you Calculator page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md), the calculator itself occupies about 2/3 of page and the 
 text (`Let's do some math`) is aligned with the page title (`Math Magicians`). Please kindly fix that.

File: src/App.js [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
+          </Route>
+          <Route exact path="/quote">
+            <Quote />
```
Comment (Mihndim2020) [inline src/App.js:24]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", Please make the quote on the `Quote` page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md), the quote is centered aligned not left aligned.  Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @AlaaAlsalem , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Clean and organized code.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (AlaaAlsalem) [reply src/App.js:21]: Sure, thanks for your feedback

File: src/App.js [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
-        <Calculater />
+        <Navbar />
+        <Switch>
+          <Route exact path="/">
 
+            <Home />
+
+          </Route>
+          <Route exact path="/calculator">
+            <Calculator />
```
Comment (Mihndim2020) [inline src/App.js:21]: - [OPTIONAL] The text(`Let's do some math!`) should be aligned with the page title(`Math Magicians`) and the caculator should occupy about 2/3 of the page, please kindly fix that. As a front-end dev, it is a very important that you stick to the design as much as possible.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @AlaaAlsalem, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**